js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```
@@ -2986,4 +2985,44 @@ customValue=Custom value
 termsAndConditionsUserAttribute=Terms and conditions accepted timestamp
 realmOverridesDescription= Realm overrides allow you to specify translations that will take effect for the entire realm. These translations will override any translation specified by a theme.
 addTranslation=Add translation
-effectiveMessageBundlesDescription=An effective message bundle is the set of translations for a given language, theme, and theme type. It also takes into account any realm overrides, which will take precedence.
\ No newline at end of file
+effectiveMessageBundlesDescription=An effective message bundle is the set of translations for a given language, theme, and theme type. It also takes into account any realm overrides, which will take precedence.
+selectAccessTokenSignatureAlgorithm=Select access token signature algorithm
+selectIdTokenSignatureAlgorithm=Select Id token signature algorithm
+selectIdTokenEncryptionKeyManagementAlgorithm=Select Id token encryption key management algorithm
+selectIdTokenEncryptionContentEncryptionAlgorithm=Select Id token encryption content encryption algorithm
+selectUserInfoSignedResponseAlgorithm=Select user info signed response algorithm
+selectUserInfoResponseEncryptionKeyManagementAlgorithm=Select user info response encryption key management algorithm
+selectUserInfoResponseEncryptionContentEncryptionAlgorithm=Select user info response encryption content encryption algorithm
+selectRequestObjectSignatureAlgorithm=Select request object signature algorithm
+selectRequestObjectEncryption=Select request object encryption
+selectRequestObjectEncoding=Select request object encoding
+selectRequestObjectRequired=Select request object required
+selectAuthorizationSignedResponseAlgorithm=Select authorization signed response algorithm
+selectAuthorizationEncryptedResponseAlg=Select authorization encrypted response algorithm
+selectAuthorizationEncryptedResponseEnc=Select authorization encrypted response enc
```

The `authorization_encrypted_response_enc` and `authorization_encrypted_response_alg` come from [OIDC spec](https://openid.net/specs/openid-financial-api-jarm.html#client-metadata), thus I'd rather keep them named consistently for user's sake: either
```suggestion
selectAuthorizationEncryptedResponseAlg=Select authorization encrypted response alg
selectAuthorizationEncryptedResponseEnc=Select authorization encrypted response enc
```
or
```suggestion
selectAuthorizationEncryptedResponseAlg=Select authorization encrypted response algorithm (alg)
selectAuthorizationEncryptedResponseEnc=Select authorization encrypted response encryption method (enc)
```